manifests/m/MicroSIP/MicroSIP-Lite/3.20.7/MicroSIP.MicroSIP-Lite.locale.en-US.yaml

```diff
@@ -0,0 +1,29 @@
+# Created using YamlCreate.ps1 v1.1.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: MicroSIP.MicroSIP-Lite
```

Comment:
```suggestion
PackageIdentifier: MicroSIP.MicroSIP.Lite
```

Reply:
@ItzLevvie I would like to decline your suggestion because the -Lite is the same way as the developer presents it on their homepage / download page.

Reply:
WinGet now supports X.X.X rather than just X.X; i.e. Google.Chrome.Dev, Google.Chrome.Canary, etc.

We're not following what the developers' websites are saying. That would probably be better suited for `PackageName` (as long as Control Panel shows it as such - it currently doesn't at the moment), but in this case, we're following for consistency with other packages that are going to move towards this format in the future.

Reply:
I find it somehow confusing, given that we have SemVer 2 widely adopted. But whatever.

So to use the naming as you suggested, I have to also move the manifest into `manifests/m/MicroSIP/MicroSIP/Lite/3.20.7/MicroSIP.MicroSIP.Lite.locale.en-US.yaml`, right? I would then rather edit the commit on my PC instead of directly here. Thanks in advance!

Reply:
Resolved in 0725fd5eb4a72647ddc134db1b2cbeda594beeb9